.github/workflows/notify-slack-on-release-merge.yml

```diff
@@ -0,0 +1,68 @@
+name: Slack notifications for releases/* merges
+
+# This CI job will push notifications to Slack whenever code is merged into any of the
+# release branches.
+
+on:
+  push:
+    branches:
+      - 'releases/*'
+
+jobs:
+  notify-slack:
+    name: 'Emit Slack notification(s)'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Parse event to Slack messages
+        env:
+          GITHUB_CONTEXT: ${{ toJson(github) }}
+        run: |
+          echo $GITHUB_CONTEXT | jq '.event.commits[] | [.url, .id[0:7], .author.username, .timestamp, .message] | @tsv' | sed 's/"//g' | sed 's/\\t/;/g' | sed 's/\\n/;/g' | sed 's/\\//g' | awk -F';' '{print "• <"$1"|"$2"> (<https://github.com/"$3"|"$3">, "$4") - "$5}' | sed 's/$/\\n/g' | tr -d '\n' > /tmp/foo
+          echo "SLACK_COMMITS=$(cat /tmp/foo)" >> $GITHUB_ENV
+      - name: Send custom JSON data to Slack workflow
```

Comment:
Does this use the parsed event? Why is it being parsed?

Reply:
Yes! This takes the content of the [`$GITHUB_CONTEXT`](https://docs.github.com/en/actions/learn-github-actions/contexts#github-context) JSON blob and formats it into the proper format for the Slack message (check out the contents of the #mobile-build-alerts Slack channel to see what the output looks like).